app/_src/mesh/guides/create-a-mesh-using-terraform-and-konnect.md

```diff
@@ -0,0 +1,602 @@
+---
+title: Create a Mesh using Terraform and Konnect
+---
+
+{% assign KM_VERSION = "2.9.3" %}
+{% assign TF_BETA_VERSION = "0.1.0" %}
+{% assign TF_TIME_VERSION = "0.13.0" %}
```

Comment:
Should these 2 be site config?

Reply:
`TF_BETA_VERSION` could be, but `TF_TIME_VERSION` is just a version of the time provider so not sure